pkg/gce-pd-csi-driver/node.go

```diff
@@ -66,7 +67,7 @@ func (ns *GCENodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePub
 		return nil, status.Error(codes.InvalidArgument, "NodePublishVolume Volume Capability must be provided")
 	}
 
-	notMnt, err := ns.Mounter.Interface.IsLikelyNotMountPoint(targetPath)
+	notMnt, err := ns.Mounter.Interface.IsNotMountPoint(targetPath)
```

Comment:
in tree uses the "likely" variant.  Is there a reason why we need to switch to the unlikely variant?

Reply:
If there is a mount point here its likely that its from a `NodePublish` bind-mount. the "likely" variant doesn't detect bind-mounts and therefore can't find the preexisting mount. I think it should be changed in-tree as well.

Reply:
The comment is misleading.

It doesn't detect if the root volume itself is a bind mount, ie you bind mounted a hostpath volume and that's the volume.

What the IsLikelyNotMountPoint check does is compare the filesystem of the volume with its parent.  And if they're different filesystems then it says it's a mount point.

Reply:
reverted